SECURITY.md

```diff
@@ -0,0 +1,68 @@
+# Security Policy
+As an open-source project, we understand the importance of and responsibility
+for security. This Security policy outlines our guidelines and procedures for
+ensuring the highest level of Security and trust for our users who consume the
+oneAPI Construction Kit.
+
+## Supported Versions
+We will perform patch releases for the supported [latest version][1] or create
+new releases which contain fixes for security vulnerabilities and important
+bugs.
+
+## Report a Vulnerability
+We are very grateful to the security researchers and users that report back
+security vulnerabilities. We investigate every report thoroughly.
+We strongly encourage you to report security vulnerabilities to us privately,
+before disclosing them on public forums or opening a public GitHub issue.
+Report a vulnerability to us in one of two ways:
+* Open a draft [**GitHub Security Advisory**][2]
+* Send e-mail to the following address: **security@uxlfoundation.org**.
```

Comment:
[MD034 - Bare URL used](https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md)
```suggestion
* Send e-mail to the following address: <security@uxlfoundation.org>.
```
Also, who receives e-mails sent to this address?

Reply:
At the organization level, there will be a team receiving incidents reported using the Security Advisory form. Same team should receive emails from this address.